content/en/blog/2019/dns-cert/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Istio DNS certificate management
+description: Provision and manage DNS certificates in Istio.
+publishdate: 2019-10-29
+attribution: Lei Tang (Google)
+keywords: [security, kubernetes, certificates, DNS]
+target_release: 1.4
+---
+
+The DNS certificates used by Istio control plane components (e.g., Galley, Sidecar Injector)
+are by default managed by Citadel, which is a large component, maintains its own private
+signing key, and also acts as a CA.
+
+In Istio 1.4, we introduce a new feature to securely provision and manage DNS certificates
+signed by Kubernetes CA in Istio, which has the following advantages.
+
+* This feature removes the dependency on Citadel for DNS certificates and is
+more lightweight than Citadel.
```

Comment:
What does "lightweight" mean? Any improved performance comes from the K8s CA doing the heavy lifting. Since it is not clear, I suggest just removing it.
```suggestion
```

Reply:
"lightweight" means that this feature has simpler implementation, less code, less moving parts.